README.md

```diff
@@ -162,6 +162,8 @@ grade.save { (id: Int?, grade: Grade?, error: RequestError?) in
 }
 ```
 
+**NB**: If you want to use `RequestError`, you'll need to import `KituraContracts` at the top of your swift file.
```

Comment:
We don't need this anymore since we created a public `typealias` for `KituraContracts` but it hasn't been tagged yet

Reply:
now has been tagged as `0.0.2`

Reply:
@dokun1 can you remove this section? Once that's done I think this is good to go.